_posts/Summary Script, Ep. 0 Open Source Directions: Bokeh

```diff
@@ -0,0 +1,90 @@
+This is the webinar that informs the community about what is going on with Open Source Python projects.  Today's project being highlighted is Bokeh.
```

Comment:
I think this file needs to be named as described [here](https://github.com/barryclark/jekyll-now#step-3-publish-your-first-blog-post), that is `_posts/YYYY-MM-DD-Title.md`  So this file would be named `_posts/2018-08-15-Episode-0-Bokeh.md`

Reply:
Also, right at the top of the we need the front matter. This would be here the following 4 lines:

```
---
layout: post
title: Open Source Directions Episode 0: Bokeh
---
```